specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -14011,126 +17307,134 @@
         }
       }
     },
-    "PerfMonResponse": {
-      "description": "Performance monitor API response.",
-      "type": "object",
-      "properties": {
-        "code": {
-          "description": "The response code.",
-          "type": "string"
-        },
-        "message": {
-          "description": "The message.",
-          "type": "string"
-        },
-        "data": {
-          "$ref": "#/definitions/PerfMonSet",
-          "description": "The performance monitor counters."
-        }
-      }
-    },
-    "PerfMonSample": {
-      "description": "Performance monitor sample in a set.",
+    "ProcessModuleInfo": {
+      "description": "Process Module Information.",
       "type": "object",
-      "properties": {
-        "time": {
-          "format": "date-time",
-          "description": "Point in time for which counter was measured.",
-          "type": "string"
-        },
-        "instanceName": {
-          "description": "Name of the server on which the measurement is made.",
-          "type": "string"
-        },
-        "value": {
-          "format": "double",
-          "description": "Value of counter at a certain time.",
-          "type": "number"
-        },
-        "coreCount": {
-          "format": "int32",
-          "description": "Core Count of worker. Not a data member",
-          "type": "integer"
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
         }
-      }
-    },
-    "PerfMonSet": {
-      "description": "Metric information.",
-      "type": "object",
+      ],
       "properties": {
-        "name": {
-          "description": "Unique key name of the counter.",
-          "type": "string"
-        },
-        "startTime": {
-          "format": "date-time",
-          "description": "Start time of the period.",
-          "type": "string"
-        },
-        "endTime": {
-          "format": "date-time",
-          "description": "End time of the period.",
-          "type": "string"
-        },
-        "timeGrain": {
-          "description": "Presented time grain.",
-          "type": "string"
-        },
-        "values": {
-          "description": "Collection of workers that are active during this time.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/PerfMonSample"
-          }
+        "properties": {
+          "description": "ProcessModuleInfo resource specific properties",
+          "properties": {
+            "baseAddress": {
+              "description": "Base address. Used as module identifier in ARM resource URI.",
+              "type": "string"
+            },
+            "fileName": {
```

Comment:
This is what I had to go on: https://github.com/projectkudu/kudu/blob/5e26bd0d5d81a6aa92303cea0370a7e491e63923/Kudu.Contracts/Diagnostics/ProcessModuleInfo.cs 

As you can see, there are zero comments on any of these fields.  So, I simply put in the best comment available.  Here's the wiki page describing the functionality: 
https://github.com/projectkudu/kudu/wiki/Process-Threads-list-and-minidump-gcdump-diagsession#summary

Again, no description of what the fields.  I presume they come from [System.Diagnostics.ProcessModule](https://msdn.microsoft.com/en-us/library/system.diagnostics.processmodule%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396)